css/types/image.json

```
@@ -2116,6 +2116,55 @@
               "deprecated": false
             }
           }
+        },
+        "paint": {
+          "__compat": {
+            "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/paint",
+            "description": "<code>paint()</code>",
+            "support": {
+              "chrome": {
+                "version_added": "65"
+              },
+              "chrome_android": {
+                "version_added": "65"
+              },
+              "edge": {
+                "version_added": false
+              },
+              "firefox": {
+                "version_added": false
+              },
+              "firefox_android": {
+                "version_added": false
+              },
+              "ie": {
+                "version_added": false
+              },
+              "opera": {
+                "version_added": "52"
+              },
+              "opera_android": {
+                "version_added": "52"
```

For Opera Android, the rule is not "Chrome - 13". 
The first version that includes Chrome 65 would be 47. See https://github.com/mdn/browser-compat-data/blob/master/browsers/opera_android.json#L234
```suggestion
                "version_added": "47"
```